VideoLocker/src/main/java/org/edx/mobile/util/AppStoreUtils.java

```diff
@@ -0,0 +1,60 @@
+package org.edx.mobile.util;
+
+import android.content.ActivityNotFoundException;
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+import android.view.View;
+import android.widget.Toast;
+
+import org.edx.mobile.BuildConfig;
+import org.edx.mobile.R;
+
+/**
+ * Utility class for interacting with an app store, or the
+ * Play Store specifically.
+ */
+public final class AppStoreUtils {
+    // Make this class non-instantiable
+    private AppStoreUtils() {
+        throw new UnsupportedOperationException();
+    }
+
+    /**
+     * Open an app store to display the app.
+     *
+     * @param context A Context for starting the new Activity.
+     */
+    public static void openAppInAppStore(final Context context) {
+        Intent intent = new Intent(Intent.ACTION_VIEW);
+        intent.addCategory(Intent.CATEGORY_BROWSABLE);
+        intent.setData(Uri.parse("market://details?id=" + BuildConfig.APPLICATION_ID));
+        try {
+            context.startActivity(intent);
+        } catch (ActivityNotFoundException e) {
+            intent.setData(Uri.parse("https://play.google.com/store/apps/details?id=" +
+                            BuildConfig.APPLICATION_ID));
```

Comment:
@BenjiLee, @bguertin: Do we want to have a config for third party customizations to specify their own app store URI, in case their preferred distribution channel is not the Play Store? The `market://` URI can be presumably handled by any app store, but we might prefer to directly try a specific one first. Similarly, if no app store is available, then the web page that we open could be customized to be something other than the Play Store.